plugins/events-backend-module-http/README.md

```diff
@@ -0,0 +1,38 @@
+# events-backend-module-http
+
+Welcome to the events-backend-module-http backend plugin!
+
+This plugin is a module for the event-backend backend plugin
+and extends it with an `HttpEndpointEventPublisher`.
+
+This event publisher will allow you to receive events via
+HTTP endpoints `POST /api/events/{topic}` and will publish these
+to the used event bus.
+
+The created endpoints depend on your configuration:
+
+```yaml
+events:
+  publishers:
+    http:
+      topics:
+        - bitbucketCloud
+        - github
+        - whatever
+```
+
+Only those topics added to the configuration will result in
+an available endpoint.
+
+The example above would result in the following endpoints:
+
+```
+POST /api/events/bitbucketCloud
+POST /api/events/github
+POST /api/events/whatever
```

Comment:
Just noting that this hard codes external endpoint names to "internal" (consumer-side) topic names. This is probably OK, but wanted to mention it that the scope of these strings is large. Made me wonder if something like 

```yaml
topics:
  - name: ghe
     targetTopic: githubv2
```

or whatever could ever start to feel needed

Reply:
I think if it gets a use case to have a different topic used at the endpoint vs internally, we could add this as a feature later.

A the moment, I don't really foresee it though.

These endpoints you would usually use as destination at webhook subscriptions, etc. There is no specific need to have another name.

Maybe if you want to rename the internal topic but don't want to update the external uses of it 🤔
Using alternative subscriber implementations shouldn't be an issue.